tests/bug-1005.sil

```diff
@@ -0,0 +1,4 @@
+\begin{document}
+\script[src=packages/svg]
+\svg[width=20%pw, src=./tests/bug-1005.svg]
```

Comment:
In the interest of making visual checks easy, I would use most of the test canvas now A6):

```suggestion
\svg[width=100%lw, src=./tests/bug-1005.svg]
```

Alernatively specifying an absolute size in `pt` units (just less than the expected line width) might be better

```suggestion
\svg[width=265pt, src=./tests/bug-1005.svg]
```